### DISCUSSION TITLE
model-viewer doesn't load models offline ?

### DISCUSSION THREAD

**Discussion by HalanoSiblee** [google/model-viewer#4036] [Q&A]
### Model viewer offline supp!
Hi
I'm using Model-viewer mini js from unpkg
my problem is That model-viewer does not load model from my local disk
`<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>`
`<model-viewer camera-controls  src="Astronaut.glb"></model-viewer>`
it's only load model if I src it from **modelviewer.dev**

**Comment by elalish** [comment]
I'd guess the problem is your web server; you need to serve any file you're hoping to load, including those on your local machine. You might find https://glitch.com/ to be an even more convenient way to work, as they support 3D assets natively.
